lib/internal/process/stdio.js

```diff
@@ -5,110 +5,118 @@ exports.setup = setupStdio;
 function setupStdio() {
   var stdin, stdout, stderr;
 
-  process.__defineGetter__('stdout', function() {
-    if (stdout) return stdout;
-    stdout = createWritableStdioStream(1);
-    stdout.destroy = stdout.destroySoon = function(er) {
-      er = er || new Error('process.stdout cannot be closed.');
-      stdout.emit('error', er);
-    };
-    if (stdout.isTTY) {
-      process.on('SIGWINCH', function() {
-        stdout._refreshSize();
-      });
+  Object.defineProperty(process, 'stdout', {
+    configurable: false,
+    enumerable: true,
```

Comment:
is `configurable: false, enumerable: true` the default for `__defineGetter__`? i.e. this isn't breaking is it?